gpkit/keydict.py

```diff
@@ -170,3 +173,8 @@ def __getitem__(self, key):
     def __setitem__(self, key, value):
         "Assigns the key itself every time."
         KeyDict.__setitem__(self, key, None)
+
+
+class FastKeyDict(KeyDict):
+    "FsatKeyDicts are KeyDicts that don't map keys, only collate arrays"
```

Comment:
and lol, now you're ready to read some aviation forecasts. 

https://www.aviationweather.gov/products/fa/boston